tasks/Dockerfile

```diff
@@ -19,7 +21,10 @@ RUN pip install numpy==1.22  # Numpy must be installed before GDAL (in requireme
 COPY ./tasks/requirements.txt /tasks/requirements.txt
 WORKDIR /tasks
 RUN pip install -r requirements.txt
-COPY ./tasks /tasks
 
+# TODO This did not exactly work:
```

Comment:
Follow up on this. Maybe cache with shared volume as we did for causal recommender.